packages/perseus-core/src/parse-perseus-json/regression-tests/item-data/radio-with-null-widgets.json

```diff
@@ -0,0 +1,2 @@
els":[],"caption":"","alt":"A class in 1895","backgroundImage":{"url":"https://cdn.kastatic.org/ka-content-images/2cd757f8be2af8caa65c5e9c2c8e491f38a504a4.jpg","width":1024,"height":782,"top":0,"scale":null,"bottom":null,"left":null},"static":false,"range":[[0,10],[0,10]],"box":[1024,782]}},"radio 1":{"type":"radio","static":false,"graded":true,"alignment":"default","id":null,"key":null,"version":{"major":1,"minor":0},"options":{"choices":[{"content":"The Prussian Model of Education continues to influence how many American public schools are run today.\n","clue":"**This statement **is** likely true, making it the best choice.** The text first explains how the idea of conformity in schools came from Prussia. Then, the text points out how “today, many American public schools still expect students to learn the same skills, at the same time, in the same ways.” From this, you can infer that the Prussian model continues to influence American public schools today. ","correct":true,"isNoneOfTheAbove":false,"widgets":null},{"content":"The Prussian Model of Education has little influence on the way that American public schools are run today.\n","clue":"This statement isn’t likely true. After explaining how the idea of conformity in schools came from Prussia, the text points out how “today, many American public schools still expect students to learn the same skills, at the same time, in the same ways.” So, you can infer that the Prussian focus on conformity still influences American public schools.\n","correct":false,"isNoneOfTheAbove":false,"widgets":null},{"content":"There have been many people fighting to eliminate the focus on conformity in American public schools.\n","clue":"This statement isn’t likely true. The text discusses how American schools focus on conformity, and points out how “studies show that this focus on conformity can cause particular problems for certain students.” But the text doesn’t mention many people fighting to eliminate schools’ focus on conformity.\n","correct":false,"isNoneOfTheAbove":false,"widgets":null},{"content":"There have been many people fighting to impose new rules within the American public school system.","clue":"This statement isn’t likely true. The text doesn’t mention people fighting to impose new rules. \n","correct":false,"isNoneOfTheAbove":false,"widgets":null}],"noneOfTheAbove":false,"hasNoneOfTheAbove":false,"countChoices":false,"deselectEnabled":false,"randomize":false,"multipleSelect":false,"onePerLine":false,"displayCount":null}}},"replace":null,"metadata":null,"images":{}},"hints":[],"answerArea":{"type":"","static":false,"graded":false,"alignment":"","id":null,"key":null,"version":null,"zTable":false,"chi2Table":false,"tTable":false,"calculator":false,"periodicTable":false,"periodicTableWithKey":false,"financialCalculatorMonthlyPayment":false,"financialCalculatorTotalAmount":false,"financialCalculatorTimeToPayOff":false,"options":null},"_multi":null,"itemDataVersion":{"major":0,"minor":1},"answer":null}
```

Comment:
Is there a reason this didn't get formatted? Feels like it would be nicer if it were across multiple lines.

Reply:
Good call!